### PR TITLE
feat: min_assets support when withdraw

### PR DESCRIPTION
**Pull request by nowooj**
## Summary of changes

<Describe summary of major changes here>

## Report of required housekeeping

- [ ] Github issue OR spec proposal link
- [ ] Wrote tests
- [ ] Added a relevant changelog entry to CHANGELOG.md

----

## (FOR ADMIN) Before merging

- [ ] Added appropriate labels to PR
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" (coding standards)
- [ ] Confirm added tests are consistent with the intended behavior of changes
- [ ] Ensure all tests pass

### CODE REVIEW

**Review by psy2848048 — APPROVED** [review]

File: contracts/dezswap_pair/README.md [hunk 1]
```diff
@@ -29,11 +29,9 @@ When providing liquidity from a smart contract, the most important thing to keep
 
 > Note before executing the `provide_liqudity` operation, a user must allow the contract to use the liquidity amount of asset in the token contract.
 
-#### Slipage Tolerance
+#### Min Assets
 
-If a user specify the slipage tolerance at provide liquidity msg, the contract restricts the operation when the exchange rate is dropped more than the tolerance.
-
-So, at a 1% tolerance level, if a user sends a transaction with 200 UST and 1 ASSET, amountUSTMin should be set to e.g. 198 UST, and amountASSETMin should be set to .99 ASSET. This means that, at worst, liquidity will be added at a rate between 198 ASSET/1 UST and 202.02 UST/1 ASSET (200 UST/.99 ASSET).
+If a user specify the `min_assets` at `withdraw_liquidity` msg, the contract restricts the operation when the returned assets are less than than the min assets.
```
Comment (jhlee-young) [inline contracts/dezswap_pair/README.md]: typo `than` twice

File: contracts/dezswap_pair/README.md [hunk 2]
```diff
@@ -95,6 +93,36 @@ So, at a 1% tolerance level, if a user sends a transaction with 200 UST and 1 AS
   ```
 
 - Withdraw Liquidity (must be sent to liquidity token contract)
+
+  1. Without Min Assets
```
Comment (jhlee-young) [inline contracts/dezswap_pair/README.md]: The below JSON seems “with min assets” 🤔

File: contracts/dezswap_pair/README.md (hunk 1 above)
Comment (nowooj) [reply contracts/dezswap_pair/README.md]: https://github.com/dezswap/dezswap-contracts/pull/6/commits/cb1a80e787d673626050ca8b09aedcc9b573df75

File: contracts/dezswap_pair/README.md (hunk 2 above)
Comment (nowooj) [reply contracts/dezswap_pair/README.md]: https://github.com/dezswap/dezswap-contracts/pull/6/commits/cb1a80e787d673626050ca8b09aedcc9b573df75

**Review by jhlee-young — APPROVED** [review]
LGTM

**Pull request merged**